<ui version="4.0" >
 <class>about_Dialog</class>
 <widget class="QDialog" name="about_Dialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>559</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>559</width>
    <height>525</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <family>Liberation Sans</family>
   </font>
  </property>
  <property name="windowTitle" >
   <string>About FreeSSM</string>
  </property>
  <widget class="QLabel" name="progversion_label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>7</y>
     <width>350</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>FreeSSM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_pushButton" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>482</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="main_tabWidget" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>80</y>
     <width>501</width>
     <height>374</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="tabShape" >
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="about_tab" >
    <attribute name="title" >
     <string>About</string>
    </attribute>
    <widget class="QLabel" name="aboutcontent_label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>12</y>
       <width>477</width>
       <height>324</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>FreeSSM is a free and easy to use diagnostic and adjustment tool for SUBARU® vehicles. It currently supports the models LEGACY®, LIBERTY®, OUTBACK®, BAJA®, IMPREZA®, FORESTER® and TRIBECA® starting with model year 1999 and provides access to the engine and transmission control units.

The use of this program is AT YOUR OWN RISK. The author is NOT LIABLE FOR ANY EFFECTS caused by usage, including UNEXPECTED VEHICLE BEHAVIOR or DAMAGES.
This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of FUNCTIONALITY ON YOUR VEHICLE, MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.

PLEASE NOTE:
This program is NOT A PRODUCT OF FUJI HEAVY INDUSTRIES LTD. OR ANY SUBARU®-ASSOCIATED COMPANY. It is a free reengineering project which is not contributed, provided or supported by any company in any way.</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="features_tab" >
    <attribute name="title" >
     <string>Features</string>
    </attribute>
    <widget class="QLabel" name="models_label" >
     <property name="geometry" >
      <rect>
       <x>195</x>
       <y>19</y>
       <width>291</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>LEGACY® / LIBERTY®, OUTBACK®, BAJA®, IMPREZA®, FORESTER®, TRIBECA®</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="modelstitle_label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>18</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text" >
      <string>Models (since MY1999):</string>
     </property>
    </widget>
    <widget class="QLabel" name="controlunits_label" >
     <property name="geometry" >
      <rect>
       <x>195</x>
       <y>64</y>
       <width>281</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Engine, Transmission</string>
     </property>
    </widget>
    <widget class="QLabel" name="dtcsinfo_label" >
     <property name="geometry" >
      <rect>
       <x>288</x>
       <y>96</y>
       <width>191</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>(Non-OBD / OBD2 / CC)</string>
     </property>
    </widget>
    <widget class="QLabel" name="nrofActuatortests_label" >
     <property name="geometry" >
      <rect>
       <x>195</x>
       <y>192</y>
       <width>45</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="nrofadjustmentvalues_label" >
     <property name="geometry" >
      <rect>
       <x>195</x>
       <y>160</y>
       <width>45</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="systemtestsinfo_label" >
     <property name="geometry" >
      <rect>
       <x>250</x>
       <y>192</y>
       <width>231</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>(Actuators / Others) (Engine only)</string>
     </property>
    </widget>
    <widget class="QLabel" name="adjustmentsinfo_label" >
     <property name="geometry" >
      <rect>
       <x>250</x>
       <y>160</y>
       <width>231</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>(Engine / Transmission)</string>
     </property>
    </widget>
    <widget class="QLabel" name="controlunitstitle_label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>64</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text" >
      <string>Control Units:</string>
     </property>
    </widget>
    <widget class="QLabel" name="systemteststitle_label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>192</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text" >
      <string>System Tests:</string>
     </property>
    </widget>
    <widget class="QLabel" name="nrofsupportedDTCs_label" >
     <property name="geometry" >
      <rect>
       <x>195</x>
       <y>96</y>
       <width>85</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="dtcstitle_label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>96</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text" >
      <string>Data Trouble Codes:</string>
     </property>
    </widget>
    <widget class="QLabel" name="adjustmentstitle_label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>160</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text" >
      <string>Adjustment Values:</string>
     </property>
    </widget>
    <widget class="QLabel" name="mbsswsinfo_label" >
     <property name="geometry" >
      <rect>
       <x>288</x>
       <y>128</y>
       <width>191</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>(Data / Switches)</string>
     </property>
    </widget>
    <widget class="QLabel" name="nrofsupportedMBsSWs_label" >
     <property name="geometry" >
      <rect>
       <x>195</x>
       <y>128</y>
       <width>54</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mbstitle_label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>128</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text" >
      <string>Measuring Blocks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="infoobd2_label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>315</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>FreeSSM is NO OBD-2 scan tool !</string>
     </property>
    </widget>
    <widget class="QLabel" name="languagestitle_label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>224</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text" >
      <string>Program languages:</string>
     </property>
    </widget>
    <widget class="QLabel" name="languages_label" >
     <property name="geometry" >
      <rect>
       <x>195</x>
       <y>224</y>
       <width>281</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>English, German</string>
     </property>
    </widget>
    <widget class="QLabel" name="supportedOStitle_label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>256</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text" >
      <string>Operating Systems:</string>
     </property>
    </widget>
    <widget class="QLabel" name="supportedOS_label" >
     <property name="geometry" >
      <rect>
       <x>195</x>
       <y>256</y>
       <width>281</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Linux/X11, Windows</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="credits_tab" >
    <attribute name="title" >
     <string>Credits</string>
    </attribute>
    <widget class="QLabel" name="author_label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>15</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Written by Comer352l (Copyright © 2008-2009)</string>
     </property>
    </widget>
    <widget class="QLabel" name="email_label" >
     <property name="geometry" >
      <rect>
       <x>35</x>
       <y>40</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>&lt;a href="mailto:comer352l@googlemail.com">Comer352l@googlemail.com&lt;/a></string>
     </property>
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
     </property>
    </widget>
    <widget class="QLabel" name="homepageURL_label" >
     <property name="geometry" >
      <rect>
       <x>170</x>
       <y>191</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>&lt;a href="http://FreeSSM.berlios.de/">http://FreeSSM.berlios.de&lt;/a></string>
     </property>
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
     </property>
    </widget>
    <widget class="QLabel" name="gplinfo_label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>222</y>
       <width>477</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
You should have received a copy of the GNU General Public License along with this program.  If not, see &lt;http://www.gnu.org/licenses/>.</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="homepagetitle_label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>190</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Project homepage:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>160</y>
       <width>471</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Background picture Copyright © Antoine Giniès</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>70</y>
       <width>485</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Based on the Qt4-Framework (Copyright © Nokia Corporation)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>100</y>
       <width>481</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Many icons are taken from the Oxygen and Chrytstal icon themes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>130</y>
       <width>471</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Truetype font "Liberation Sans" Copyright © Red Hat, Inc.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="licence_tab" >
    <attribute name="title" >
     <string>Licence</string>
    </attribute>
    <widget class="QTextBrowser" name="licence_textBrowser" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>477</width>
       <height>324</height>
      </rect>
     </property>
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="changelog_tab" >
    <attribute name="title" >
     <string>Changelog</string>
    </attribute>
    <widget class="QTextBrowser" name="changelog_textBrowser" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>477</width>
       <height>325</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="lineWrapColumnOrWidth" >
      <number>0</number>
     </property>
     <property name="overwriteMode" >
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::TextSelectableByMouse</set>
     </property>
     <property name="openLinks" >
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="subtitle_label" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>48</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text" >
    <string>A free diagnostic and adjustment tool for SUBARU® vehicles</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="trademarksInfo_label" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>460</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>All trademarks are property of Fuji Heavy Industries Ltd. or their respective owners.</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>main_tabWidget</tabstop>
  <tabstop>close_pushButton</tabstop>
  <tabstop>licence_textBrowser</tabstop>
  <tabstop>changelog_textBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/FreeSSM.qrc" />
 </resources>
 <connections/>
</ui>
